From 84dbb858fc56ae8572a7468b8a2fd163bb4794a8 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:01:41 -0400
Subject: [usb] core/hcd: removed braces for return statements

Message-id: <1410887463-20674-30-git-send-email-dzickus@redhat.com>
Patchwork-id: 95370
O-Subject: [RHEL7 PATCH 029/591] drivers: usb: core: hcd: removed braces for return statements
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 070104966cde138627fea8257ef278f7a0d1379b
Author: Matthias Beyer <mail@beyermatthias.de>
Date:   Sat Oct 5 18:02:09 2013 +0200

    drivers: usb: core: hcd: removed braces for return statements

    Signed-off-by: Matthias Beyer <mail@beyermatthias.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/core/hcd.c b/drivers/usb/core/hcd.c
index b98f3d9..2a538ce 100644
--- a/drivers/usb/core/hcd.c
+++ b/drivers/usb/core/hcd.c
@@ -1121,18 +1121,18 @@ long usb_calc_bus_time (int speed, int is_input, int isoc, int bytecount)
  case USB_SPEED_LOW:  /* INTR only */
   if (is_input) {
    tmp = (67667L * (31L + 10L * BitTime (bytecount))) / 1000L;
-   return (64060L + (2 * BW_HUB_LS_SETUP) + BW_HOST_DELAY + tmp);
+   return 64060L + (2 * BW_HUB_LS_SETUP) + BW_HOST_DELAY + tmp;
   } else {
    tmp = (66700L * (31L + 10L * BitTime (bytecount))) / 1000L;
-   return (64107L + (2 * BW_HUB_LS_SETUP) + BW_HOST_DELAY + tmp);
+   return 64107L + (2 * BW_HUB_LS_SETUP) + BW_HOST_DELAY + tmp;
   }
  case USB_SPEED_FULL: /* ISOC or INTR */
   if (isoc) {
    tmp = (8354L * (31L + 10L * BitTime (bytecount))) / 1000L;
-   return (((is_input) ? 7268L : 6265L) + BW_HOST_DELAY + tmp);
+   return ((is_input) ? 7268L : 6265L) + BW_HOST_DELAY + tmp;
   } else {
    tmp = (8354L * (31L + 10L * BitTime (bytecount))) / 1000L;
-   return (9107L + BW_HOST_DELAY + tmp);
+   return 9107L + BW_HOST_DELAY + tmp;
   }
  case USB_SPEED_HIGH: /* ISOC or INTR */
   /* FIXME adjust for input vs output */
-- 
1.7.1